### ISSUE TITLE
nethereumUnityInstance is not defined

### ISSUE THREAD

**Issue by Rocktome** [Nethereum/Nethereum.Unity.Webgl#5]
Uncaught (in promise) ReferenceError: nethereumUnityInstance is not defined
    at _EnableEthereum

**Comment by juanfranblanco** [comment]
Check https://github.com/Nethereum/Nethereum.Unity.Webgl/issues/1 and https://github.com/Nethereum/Nethereum.Unity.Webgl/issues/2

**Comment by Rocktome** [comment]
Thank you very much

**Comment by BestStream** [comment]
> Check #1 and #2

You are right - it helped. But it would be easier if you added a template to the project, then it would work out of the box
